qemu/tests/qsd_install.py

```diff
@@ -0,0 +1,54 @@
+"""QSD installation test"""
+
+from provider.qsd import QsdDaemonDev, add_vubp_into_boot
+from virttest import env_process
+from virttest.tests import unattended_install
+
+
+def run(test, params, env):
+    """
+    Test installation test .
+    Steps:
+        1) Run QSD with one export vhost-user-blk.
+        2) Boot VM with vhost-user-blk-pci device and install media.
+        3) Wait for installation finish
+        4) Login guest and run check command
+    """
+
+    logger = test.log
+    qsd = None
+
+    try:
+        qsd_name = params["qsd_namespaces"]
+        qsd = QsdDaemonDev(qsd_name, params)
+        qsd.start_daemon()
+        img = params["qsd_images_qsd1"]
+        add_vubp_into_boot(img, params, 6)
+
+        logger.debug("unattended_install start")
+        unattended_install.run(test, params, env)
+        logger.debug("Finish installation")
+        vm = env.get_vm(params.get("main_vm"))
+        vm.destroy()
+        logger.debug("Reboot...")
+        params["cdroms"] = ""
+        params["start_vm"] = "yes"
+        params["cdrom_unattended"] = ""
+        params["kernel"] = ""
+        params["initrd"] = ""
+        params["boot_once"] = "c"
+        params["kernel_params"] = ""
+        env_process.preprocess_vm(test, params, env, params.get("main_vm"))
+        vm = env.get_vm(params.get("main_vm"))
```

Comment:
Suggest using the following two statements. I think no need to import env_process. 

vm.create(params=params)
vm.verify_alive()

with the update, the result was passed. I think we can remove line 4, 41 and 42. If I was wrong, please correct me. Thanks.
(1/1) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.qsd_install.q35: PASS (662.11 s)

Reply:
updated